### PR TITLE
Proofread homepage

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by andrewconner** [comment]
@2is10 You know english.

File: kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html [hunk 1]
```diff
@@ -123,7 +123,7 @@ <h1 class="duration3">Never lose your research again</h1>
                         </div>
                         <div class="description">
                             <h3>Keeping and finding made easy</h3>
-                            <p>Easily collect everything that matter to you in one click . Quickly find it later in your very own Kifi Library, or when searching your favorite search engine</p>
+                            <p>Easily collect everything that matter to you in one click. Quickly find it later in your very own Kifi Library, or when searching your favorite search engine</p>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html:126]: matter → matters

Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html:126]: when searching → with
(suggestion)

Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html:126]: End paragraph with a period for consistency with others.

File: kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html [hunk 2]
```diff
@@ -143,7 +143,7 @@ <h1 class="duration3">Never lose your research again</h1>
                         <div class="description">
                             <h3>Share and collaborate on pages of interest</h3>
                             <p>Send a page or even start a private discussion on any page
-    that inspire you, with a friend or a group of friends</p>
+    that inspire you, with a friend or a group of friends.</p>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/marketing/landingNew.scala.html:146]: inspire → inspires

**Pull request merged**